api/preprints/views.py

```diff
@@ -0,0 +1,197 @@
+            display_name='preprint'
+        )
+        if not node.is_preprint and self.request.method != 'POST':
+            raise NotFound
+
+        return node
+
+
+class PreprintList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin):
+    """Preprints that represent a special kind of preprint node. *Read Only*.
+
+    Paginated list of preprints ordered by their `date_created`.  Each resource contains a representation of the
+    preprint.
+
+    ##Preprint Attributes
+
+    Many of these preprint attributes are the same as node, with a few special fields added in.
+
+    OSF Preprint entities have the "preprint" `type`.
+
+        name                            type                  description
+        ====================================================================================
+        title                           string                title of preprint, same as its project or component
+        abstract                        string                description of the preprint
+        date_created                    iso8601 timestamp     timestamp that the preprint was created
+        date_modified                   iso8601 timestamp     timestamp when the preprint was last updated
+        tags                            array of strings      list of tags that describe the node
+        subjects                        array of dictionaries list ids of Subject in the PLOS taxonomy. Dictrionary, containing the subject text and subject ID
+        provider                        string                original source of the preprint
+        doi                             string                bare DOI for the manuscript, as entered by the user
+
+
+    ##Relationships
+
+    ###Primary File
+    The file that is designated as the preprint's primary file, or the manuscript of the preprint.
+
+    ###Files
+    Link to list of files associated with this node/preprint
+
+    ###Contributors
+    Link to list of contributors that are affiliated with this institution.
+
+    ##Links
+
+    - `self` -- Preprint detail page for the current preprint
+    - `html` -- Project on the OSF corresponding to the current preprint
+    - `doi` -- URL representation of the DOI entered by the user for the preprint manuscript
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+    ##Query Params
+
+    + `page=<Int>` -- page number of results to view, default 1
+
+    + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
+
+    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, `provider`, and `subjects`
+    Most are string fields and will be filtered using simple substring matching.
+
+    #This Request/Response
+    """
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.NODE_BASE_READ]
```

Comment:
That's a good question- and a mite tricky, as at last check you could see the full data for a preprint by asking for `v2/nodes/<preprint_guid>`.

(Eg preprints were under both endpoints, so having `nodes` alone would give access to data about the preprint node too)

My instincts are as follows:
- Scopes are about data, and preprints have a different serializer. Give them their own CoreScope to reflect this difference.
- For now bundle them in existing endpoints
- *The biggest thing to protect separately would be the preprints write scope*: letting someone edit a page on your behalf should not automatically allow them to submit that for irrevocable(ish) publication.
- As for read scope, most data overlaps with nodes. The main edge case I see where we might want to protect preprint data separately as confidential is if we create a category of preprints that were submitted, but not accepted.
  - Such functionality has been marked explicitly out of scope for the 1.0 release at the end of the month, so protecting read this way is not an immediate priority.

Regardless, all these permissions will end up in osf.full_read/write... at some point it's worth revisiting more granular scopes.